test/journal_test.go

```diff
@@ -89,6 +89,92 @@ func TestJournalCrSimple(t *testing.T) {
 	)
 }
 
+// bob creates a conflicting file while running the journal, and then
+// hits another conflict while trying to put the resolution.
+// Regression test for KBFS-1590.
+func TestJournalCrConflictOnPut(t *testing.T) {
```

Comment:
The test is giving tons of problems under fuse/dokan -- it looks like it's because there is some timing issue around cleaning up blocks after the failed resolution, and it's hard to predict ahead of time exactly how many GetRange calls there will be.  So I'm going to nuke it, sadly.